ember/ember.d.ts

```diff
@@ -1039,6 +1041,13 @@ declare module Ember {
         static create<T extends {}>(args?: {}): T;
 
         /**
+        Creates an instance of the class.
+        @method create
+        @param [args] - A hash containing values with which to initialize the newly instantiated object.
+        **/
+        create<T extends {}>(args?: {}): T;

```

Comment:
Actually I think this was a bug in ReSharper (but not the TypeScript compiler). I'm removing this now.